bp_top/test/tb/bp_tethered/Makefile.vcs

```diff
@@ -47,6 +47,8 @@ $(SIM_DIR)/bootrom.mem: $(SIM_DIR)/bootrom.riscv
 	$(RISCV_OBJCOPY) -O verilog --reverse-bytes=8 --verilog-data-width=8 $< $@
 
 NBF_INPUTS ?= --ncpus=$(NCPUS) --ucode=cce_ucode.mem
+NBF_INPUTS += --domains=$(DOMAINS_P)
```

Comment:
same comment as above, probably unnecessary

Reply:
Requested for clarification above.